samples/app-identity-linking-with-sso/nodejs/index.js

```diff
@@ -343,7 +343,7 @@ server.post('/getFbAccessToken', function (req, res) {
   });
 });
 
-server.post('/getGoogleAccessToken', function (req, res) {
+server.post('/getGoogleDetailsOauth', function (req, res) {
```

Comment:
Are we getting Oauth or just Google account user details?

Reply:
we are getting google account details following Outh flow 
updated route as per suggestion